spec/system/comments/user_edits_a_comment_spec.rb

```diff
@@ -19,13 +19,12 @@
   def assert_updated
     expect(page).to have_css("textarea")
     expect(page).to have_text("Editing comment")
-    fill_in "text-area", with: new_comment_text
+    find("textarea").send_keys(new_comment_text)
```

Comment:
This may have been the root cause of the flaky spec. This now append to the end of the textarea instead of trying to replace it.